src/main/java/io/github/darealturtywurty/ancientology/core/materials/ToolMaterials.java

```diff
@@ -0,0 +1,60 @@
+package io.github.darealturtywurty.ancientology.core.materials;
+
+import java.util.function.Supplier;
+import io.github.darealturtywurty.ancientology.core.init.ItemInit;
+import net.minecraft.world.item.Items;
+import net.minecraft.world.item.Tier;
+import net.minecraft.world.item.crafting.Ingredient;
+
+public enum ToolMaterials implements Tier {
+   FLINT(0, 59, 2.0F, 0.0F, 15, () -> {
+	   return Ingredient.of(Items.FLINT);
+   }),
+   BONE(0, 59, 2.0F, 0.0F, 15, () -> {
+	   return Ingredient.of(Items.BONE);
+	   }),
+   BRONZE(1, 131, 4.0F, 1.0F, 5, () -> {
+      return Ingredient.of(ItemInit.BRONZE_INGOT);
+   });
+
+   private final int level;
+   private final int uses;
+   private final float speed;
+   private final float damage;
+   private final int enchantmentValue;
+   private final Supplier<Ingredient> repairIngredient;
+
+   private ToolMaterials(int level, int uses, float speed, float damage, int enchantmentValue, Supplier<Ingredient> repairIngredient) {
+      this.level = level;
+      this.uses = uses;
+      this.speed = speed;
+      this.damage = damage;
+      this.enchantmentValue = enchantmentValue;
+      this.repairIngredient = repairIngredient;
+   }
+
+   public int getUses() {
```

Comment:
Again, final fields don't really need to be private as you cant set a final field once the constructor has been called, these methods just take up space.

Reply:
No, it is needed. This method overrides something.. This is @jzimmerman4's fault for not using `@Override`, so please fix it.